frontend/src/pages/HomePage/HomePage.tsx

```diff
@@ -1,12 +1,152 @@
-import GeocacheList from "../../components/GeocacheList";
+import { useState, useEffect } from "react";
+import { getGeoCaches } from "../../api/geocaches";
+import { Container, Card, Row, Button } from "react-bootstrap";
+import "./HomePage.scss";
 
 const HomePage = () => {
+    const [geocaches, setGeocaches] = useState([]);
+
+    useEffect(() => {
+        getGeoCaches().then(geocachesResult => {
+            setGeocaches(geocachesResult.data);
+        });
+    }, []);
+
+    const name1: string = geocaches[0]["name"];
+    const time1: string = geocaches[0]["publishedDate"];
+    const name2: string = geocaches[1]["name"];
+    const time2: string = geocaches[1]["publishedDate"];
+    const name3: string = geocaches[2]["name"];
+    const time3: string = geocaches[2]["publishedDate"];
+
     return (
-        <div>
-            <h1>{"Here is the page's content"}</h1>
-            <GeocacheList/>
-        </div>
+        <>
+            <div className="screen">
+                <div className="main">
+                    <Container>
+                        <Row><div className="mx-auto" style={{ width: "auto" }}>
```

Comment:
Also, the styles should be in the scss file, not in this file

Reply:
Okay, this happens, when someone is copy pasting things from the web.